pkg/reconciler/testing/trigger.go

```diff
@@ -187,6 +187,10 @@ func WithTriggerSubscriptionReady(t *brokerv1beta1.Trigger) {
 	t.Status.MarkSubscriptionReady()
 }
 
+func WithBrokerNotReady(t *brokerv1beta1.Trigger) {
```

Comment:
nits: not ready and does not exist seems very different.

Reply:
yea, that's a good point. actually,I find an existing one that I can use. deleted this one.